docs/notebooks/negative_binomial.ipynb

```diff
@@ -18,28 +18,36 @@
    "cell_type": "markdown",
```

Comment:
<p>Can you replace PyMC3 with PyMC and put an up-to-date link?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bambinos/bambi/pull/884/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>